set.mm

```diff
@@ -645047,6 +645080,263 @@ fixed reference functional determined by this vector (corresponding to
+      WCVUFVVSAVUAVUFVUJXMZXTWDVYBVUPVVSVUFVYEVJVWAVUPVWDVWTWDVWAVVSVWDVWMWDVUE
+      VYEGUXSSUDZUWBVUCSUDZVJBCGVVNUWNUWNUXRGUOZUXTVYGVUDVYHUXRGUXSSXOZVYIVUBUW
+      BVUCSUXRGKWLXPYAUXSVVNUOZVYGVWDVYHVYDUXSVVNGSXQZVYKVUCVVOUWBSUXSVVNKWLXRY
+      AXSVAYBWCYLYGYHYJWBYEYMVRAVVRVVJVHUWCAVVRVVPUBUWNYNZVCZVVJVVRVYNVHAVVPUBU
+      WNYOUPAVVJVYMAVVJVCZUWAUWGTZVYMVYOVYPVHAUWAUWGYPUPAUXOVYPVYMVHVULUBUWNUWA
+      KYQWNYRYSYRWDWOUWAUWFUWGYTVAAUWDVFZUWBUWGTZUWBUWFXAZUWHVYQKXBZGKXCZTZVYRA
+      VYTUWDAUWNUXPKVUKXDWDAWUBUWDAGUWNWUAVUQAUWNUXPKVUKXEXFWDGKXIVAVYQVWPUWBUO
+      ZVCZUBUWNVBZVYSVYQWUDUBUWNVYQVVSVFVWPUWBAUWDVVSVWPUWBUGZAUWDVVSUTZVWEWUFW
+      UGVVNGWUGVVNVVSAVWFUWDVWGXGXHAUWDVWHVVSVWIXTXJWUGVWBWUFVWCVWDWUGVWBVFZVWP
+      UWBWUGVWPUMTZVWBWUGVWPWUGVWPUXPTVWPWGTWUGUWNUXPVVNJAUWDUXQVVSUYSXTAUWDVVS
+      XKZWTUXPWGVWPVWNXLWNXHZWDWUHVWPUWBSUDVVOUWBSUDZWUGVWBWULWUHVUFVWBWULVJZWU
+      GVUFVWBAUWDVUFVVSVYFXTWDWUHVVSVUPVUFWUMVJAUWDVVSVWBXNWUGVUPVWBAUWDVUPVVSV
+      UQXTZWDVUEWUMVXAVVOVUCSUDZVJBCVVNGUWNUWNVXCUXTVXAVUDWUOVXDVXCVUBVVOVUCSUX
+      RVVNKWLXPYAVXEVXAVWBWUOWULVXFVXEVUCUWBVVOSUXSGKWLXRYAXSVAYBWCWUHVWPVVOUWB
+      SAVVSVWBVXGUWDVXSYFXPWKYGWUGVWCVFZWUFUWIWUPUWBUWAWUPUWBUWAWUGVUTVWCAUWDVU
+      TVVSVVGXTZWDAUWDVVSVWCYKYGYHVWCWUFUWIVHWUGVWCVWPUWAUWBVVNGJWLYIVQWKWUGVWD
+      VFZUWBVWPWURUWBVWPWUGVUTVWDWUQWDZWURUWBUWAVWPWUSWUGVUSVWDAUWDVUSVVSVVFXTW
+      DWUGWUIVWDWUKWDAUWDVVSVWDYKWUGVWDUWAVWPSUDZWURUYEVWDWUTVJZWUGUYEVWDAUWDUY
+      EVVSVWSXTWDWURVUPVVSUYEWVAVJWUGVUPVWDWUNWDWUGVVSVWDWUJWDUYDWVAVYGUWAUYBSU
+      DZVJBCGVVNUWNUWNVYIUXTVYGUYCWVBVYJVYIUYAUWAUYBSUXRGJWLXPYAVYKVYGVWDWVBWUT
+      VYLVYKUYBVWPUWASUXSVVNJWLXRYAXSVAYBWCYLYGYHYJWBYEYMVRAWUEVYSVHUWDAWUEWUCU
+      BUWNYNZVCZVYSWUEWVDVHAWUCUBUWNYOUPAVYSWVCAVYSVCZUWBUWFTZWVCWVEWVFVHAUWBUW
+      FYPUPAUXNWVFWVCVHUYTUBUWNUWBJYQWNYRYSYRWDWOVYRVYSVFUWGUWFUWBUWGUWFYTYHVAU
+      VTWB $.
+  $}
+
+  ${
+    $d A a b z $.  $d A f i j z $.  $d B z $.  $d F i j $.  $d K a b x y $.
+    $d K f x y $.  $d K r s $.  $d N a $.  $d N f $.  $d a b ph z $.
+    $d f i j ph z $.  $d i j r s $.  $d i j r x y $.  $d x y z $.
+    sticksstones2.1 $e |- ( ph -> N e. NN0 ) $.
+    sticksstones2.2 $e |- ( ph -> K e. NN0 ) $.
+    sticksstones2.3 $e |- B = { a e. ~P ( 1 ... N ) | ( # ` a ) = K } $.
+    sticksstones2.4 $e |- A = { f | ( f : ( 1 ... K ) --> ( 1 ... N ) /\ A. x
+    e. ( 1 ... K ) A. y e. ( 1 ... K )
+    ( x < y -> ( f ` x ) < ( f ` y ) ) ) } $.
+    sticksstones2.5 $e |- F = ( z e. A |-> ran z ) $.
+    $( The range function on strictly monotone functions with finite domain and
+       codomain is an injective mapping onto ` K ` -elemental sets.
+       (Contributed by metakunt, 27-Sep-2024.) $)
+    sticksstones2 $p  |- ( ph -> F : A -1-1-> B ) $=
+      ( cfv wceq wcel clt vi vj vb vs vr wf cv wi wral wa wf1 crn chash cfz cpw
+      c1 co crab fveqeq2 cfn fzfid wbr eleq1w feq1 fveq1 breq12d imbi2d ralbidv
+      wb anbi12d bibi12d cab wal abeq2 spi chvarvv biimpi adantl simpld sselpwd
```

Comment:
see above

Reply:
PS: I think that MMJ2 only showed me abeq2 because it has no hypotheses, it tends to show me theorems in closed form.

Reply:
solved, see above.